CONFIGURATION.md

```diff
@@ -0,0 +1,20 @@
+# SimpleSerialShell Configuration
+## Input buffer size
+The predefined size of input buffer is 88 bytes (defined by the ``SIMPLE_SERIAL_SHELL_BUFSIZE`` identificator). It's ok in most cases, but when it's not enough, you can redefine it with build options.
+Check your IDE documentation to learn how to properly redefine build options. See examples for Arduino IDE and PlatformIO below.
+
+### Arduino IDE:
+Now Arduino IDE allows to redefine build options through definition of board parameters.
+Find the `*board_name*.build.extra_flags` parameter for your board at  `*path_to_arduino_ide*/hardware/arduino/avr/boards.txt` file and set a new limit:
+`-DSIMPLE_SERIAL_SHELL_BUFSIZE=128`.
```

Comment:
(for Arduino IDE...) I'm concerned that even if setting SIMPLE_SERIAL_SHELL_BUFSIZE this way can work, there are so many ways that it can go wrong that we should just say "avoid doing this for now using the Arduino IDE".  

For example:

- This technique basically sets a global environment variable in the Arduino IDE.  Since this variable is not in the same git repository as the rest of your application, it will be difficult to remember to set this global variable, or even that you had set it some months ago.  Perhaps you might even forget what the correct value was supposed to be.  That can be a problem when upgrading or replacing your development system.
- While you may have correctly set SIMPLE_SERIAL_SHELL_BUFSIZE for your own application, it won't work for your friends when you share your code (until they make identical changes in their environments).
- Since SIMPLE_SERIAL_SHELL_BUFSIZE is a _global_ environment variable (for a particular board), it will be used in ALL your sketches which use SimpleSerialShell.  This can break other unrelated sketches -- they suddenly run out of storage with no code changes.  When this happens -- particularly to stable code -- it's REALLY frustrating.  You don't want to do that to your friends or yourself.

I've actually had each of these kinds of issues happen to me during development.  It can be very difficult to discover what is actually the issue.

Reply:
See 9d98012a4848f1f0e6d2d99995b467cf13d24879 for my changes to CONFIGURATION.md.

Reply:
I'm OK with these changes. It's up to you.